App.tsx

```diff
@@ -219,29 +199,23 @@ const App = () => {
           <SafeAreaProvider>
             <NavigationContainer>
               <SafeAreaView style={styles.container}>
-                <Tab.Navigator initialRouteName="Home" screenOptions={{headerShown: false}}>
```

Comment:
I'm now realizing if we had the center picker navigate to a view that has this whole thing as a child, then we are going to have a consistent route data passing paradigm .. ?

Reply:
Hm no I can't imagine adding new tabs to one global navigator with the same name but different parameters would be a good idea.